src/test/java/se/sundsvall/billingpreprocessor/integration/db/InvoiceFileRepositoryTest.java

```diff
@@ -0,0 +1,96 @@
+package se.sundsvall.billingpreprocessor.integration.db;
+
+import static java.time.OffsetDateTime.now;
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+import static org.assertj.core.groups.Tuple.tuple;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static se.sundsvall.billingpreprocessor.integration.db.model.enums.InvoiceFileStatus.GENERATED;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.transaction.annotation.Transactional;
+
+import se.sundsvall.billingpreprocessor.integration.db.model.InvoiceFileEntity;
+
+/**
+ * Invoice file repository tests.
+ *
+ * @see /src/test/resources/db/testdata-junit.sql for data setup.
+ */
+@SpringBootTest
+@ActiveProfiles("junit")
+@Transactional
+@Sql(scripts = {
+	"/db/scripts/truncate.sql",
+	"/db/scripts/testdata-junit.sql"
+})
+class InvoiceFileRepositoryTest {
```

Comment:
I think this test should use flyway schemas, since it is the source of truth when it comes to db schema.